BritneyS/SFWCardsAgainstHumanity
# Selection

## What you did :question:
- Added a data structure to hold Black Card + White Card phrase selections
- Added a button that is only enabled when all required phrases are selected
- Refactoring 

## How you did it :white_check_mark:
- Added selectionLimit (based on pickNumber) to track number of selections need to complete the round
- Decreased selectionLimit by 1 whenever phrase was selected
- Configured seeSelectionButton to only be enabled when all required phrases are selected


## How to test it :microscope:
- Run the app
- Tap 'Play!'
- 'See Selection' button should be disabled and greyed out
- Select phrases according to pickNumberLabel
- Number shown in pickNumberLabel (selectionLimit) should decrease with each selection
- When all phrases are selected, 'See Selected' button should be enabled and not greyed out


## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-10-15 at 15 59 36](https://user-images.githubusercontent.com/8409475/46975066-b168b780-d093-11e8-8ce8-1e1d6cc5fd2b.png)
![simulator screen shot - iphone xr - 2018-10-15 at 15 59 47](https://user-images.githubusercontent.com/8409475/46975077-b6c60200-d093-11e8-8ad2-d97de515fb02.png)
![simulator screen shot - iphone xr - 2018-10-15 at 15 59 53](https://user-images.githubusercontent.com/8409475/46975094-bcbbe300-d093-11e8-9720-13b5c56808c3.png)
